CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+## Development workflow
+
+> **Working on your first pull request?** You can learn how from this _free_ series: [How to Contribute to an Open Source Project on GitHub](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github).
```

Comment:
Might want to change this to "New to open source?" or something. Any new contributor would be working on their first pull request in context of the project, but many contributors probably already know how to create a pull request in general.